packages/tbds-support/lib/breakpoints.scss

```diff
@@ -0,0 +1,15 @@
+$tbds-breakpoint-small: 480px;
+$tbds-breakpoint-medium: 768px;
+$tbds-breakpoint-large: 1024px;
+
+@mixin tablet {
```

Comment:
I’d really like to avoid device-driven names, as it encourages design driven by device types rather than for the Web's natural fluidity. Thoughts on naming these mixins `small`, `medium`, and `large` like the variables?

Reply:
I like that idea. How do you feel about the ascending priority? In this model all rules at the root level are for the smallest viewport size, with rules being added or modified at larger sizes. (note the absence of a @mixin for `small`).

Do you think it needs to be more robust to handle `max-width` queries instead of just `min-width`?

Reply:
> How do you feel about the ascending priority? In this model all rules at the root level are for the smallest viewport size, with rules being added or modified at larger sizes.

Yeah, I think that pairs nicely with the mental model of mobile-first. Though, now that I think about it, I feel like a `small` query would be nice for adjusting things in viewports just above the size of an average phone-like device (e.g. moving a grid from one-column to two-column).

I guess it’s also nice from just a “completeness” point of view. As a consumer of the system, it would be easy to understand the three provided variables, and each variable is used for each query mixin. Rather than trying to use the mixin for `small`, only to realize it is a provided variable, but no mixin is provided. This could help promote “obviousness”, as a feature.

Thoughts?

> Do you think it needs to be more robust to handle `max-width` queries instead of just `min-width`?

For the sake of simplicity, I think we can start super small and just run with `min-width`. We can always add functionality for other queries in the future.